Source/Router.swift

```diff
@@ -187,7 +190,18 @@ public final class Router {
             client.URLProtocol(server, didLoadData: data)
         }
         
-        client.URLProtocolDidFinishLoading(server)
+        let didFinishLoading: (NSURLProtocol) -> () = { (server) in
+            client.URLProtocolDidFinishLoading(server)
+        }
+        
+        if latency > 0 {
+            let delayTime = dispatch_time(DISPATCH_TIME_NOW, Int64(latency * Double(NSEC_PER_SEC)))
+            dispatch_after(delayTime, dispatch_get_main_queue()) {
+                didFinishLoading(server)
+            }
+        } else {
+            didFinishLoading(server)
+        }
```

Comment:
Why not just using `dispatch_after` directly? Isn't the overhead with `latency = 0` negligible?